roles/common/defaults/main.yml

```diff
@@ -31,7 +31,7 @@ common:
     disable_dns: True
     client_alive_interval: 1800
   pip_version: 8.0.2
-  setuptools_version: system
+  setuptools_version: '>=16.0,!=24.0.0,<34.0'
```

Comment:
I though we talked about upgrading setuptools passing -c <path_to_constrain_file> like we do in openstack-source role.